test/e2e/README.md

```diff
@@ -2,7 +2,9 @@
 
 ## Run E2E tests Locally
 ### Prerequisite
-Make sure a kubernetes cluster is set up and kubeconfig is under $HOME/.kube/config
+ - Make sure a kubernetes cluster(with version >= 1.13) is set up and kubeconfig is under `$HOME/.kube/config`
```

Comment:
After the feature "Raw Block Volume" and its e2e test are merged, the cluster version must be greater than or equal to 1.14.